src-db/database/sourcedata/AD_MENU.xml

```diff
@@ -4521,7 +4521,7 @@
 <!--1005400001-->  <AD_MENU_ID><![CDATA[1005400001]]></AD_MENU_ID>
 <!--1005400001-->  <AD_CLIENT_ID><![CDATA[0]]></AD_CLIENT_ID>
 <!--1005400001-->  <AD_ORG_ID><![CDATA[0]]></AD_ORG_ID>
-<!--1005400001-->  <ISACTIVE><![CDATA[Y]]></ISACTIVE>
+<!--1005400001-->  <ISACTIVE><![CDATA[N]]></ISACTIVE>
```

Comment:
If this menu element is not going to be active ever again, can it be deleted altogether? Or is there any reason for it to still exist?